lib/src/constants/config.dart

```diff
@@ -138,6 +139,10 @@ class CameraPickerConfig {
   /// 摄像机在拍摄时锁定的旋转角度
   final DeviceOrientation? lockCaptureOrientation;
 
+  /// The possible flash modes that can be set for a camera.
+  /// 可以为相机设置闪光模式
+  final FlashMode flashMode;
```

Comment:
Please reference to `preferredLensDirection` and apply the similar comments.

```suggestion
  final FlashMode preferredFlashMode ;
```

Reply:
get